tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
OK, then I think we need more opinion on this case:

Should `[FOO => true, BAR =>false]` means the same as simply `[FOO => true]` or not, when it is part of ruleset definition like PSR or PHPMigration ?